HW-0.Rmd

```diff
@@ -21,9 +21,9 @@ completion. Your analysis is now online! You can update what's online by
 repeating steps 2 and 3.
 1. Copy the URL of the resulting RPubs page.
 1. Replace the https://www.nhl.com/ URL below with the URL of your RPubs page:  
-[My RPubs Document](https://www.nhl.com/)
+[My RPubs Document](http://rpubs.com/anarciso)
```

Comment:
GitHub as a great GUI (graphical user interface) for evaluating differences in code. This is how I'll be primarily delivering feedback this semester. Also, you can write comments in Markdown, which R Markdown uses. More on Markdown later.

Note here you copied a link to your RPubs homepage and not your HW-0 output. Not a big deal as I was able to find it on your home page, but something to keep in mind if you're trying to share more substantive work with others!